### PR TITLE
Various changes

### PR DESCRIPTION
**Pull request by DreierF**
Hey,

I wanted to polish the UI a little bit and replaced the pre-rendered images by vector drawables.
The started progress no has an animation just like on the web UI :)
The view_build also has been a little bit simplified and all TextViews should now nicely align.

What do you think?

### CODE REVIEW

**Comment by DreierF** [comment]
Thanks for the reply!
OK I'll split it into multiple PRs

File: app/build.gradle [hunk 1]
```diff
@@ -61,9 +61,9 @@ ext {
             eventbus : '2.4.0',
             circleimageview : '2.0.0',
             dagger : '2.0.2',
-            mockito : '2.7.0',
+            mockito : '1.10.19',
```
Comment (dkhmelenko) [inline app/build.gradle]: @DreierF What is the reason to downgrade mockito and robolectric?

Comment (DreierF) [inline app/build.gradle]: The app originally used Version 1.10.19. I upgraded it to 2.7.0 and realized that this led to compile errors or test failures (can't remember exactly) so I reverted to the old version for now

Comment (dkhmelenko) [inline app/build.gradle]: ah, I see, I'll take a look on it later

**Pull request closed without merging**